python/rikai/types/geometry.py

```diff
@@ -102,6 +111,33 @@ class Box2d(ToNumpy, Sequence, ToDict, Drawable):
     >>> draw.rectangle(box, fill="green", width=2)
     """
 
+    def left_top(self):
+        return self.xmin, self.ymin
+
+    def aligned(
+        self, base_drawable: ComposableDrawable, algorithm: str = "left-top"
+    ):
+        (new_xmin, new_ymin) = base_drawable.left_top()
```

Comment:
I am confused.  What usecase is it for a newly aligned bbox with other object (assuming box2d)?

Reply:
It's for implementing the `ComposableDrawable` trait, doable but not useful yet, maybe it would be eventually not useful, I would consider keeping it unimplemented until we really need it.